usr/share/rear/lib/layout-functions.sh

```diff
@@ -680,26 +680,39 @@ function get_part_device_name_format() {
             part_name="${device_name}p" # append p between main device and partitions
             ;;
         (*mapper[/!]*)
+
+            # check if multipath if using the "user_friendly_names" by default in the current configuration.
+            user_friendly_names=$(echo "show config" | multipathd -k | awk '/user_friendly_names/ { gsub("\"","") ; print $2 }' | head -n 1 )
```

Comment:
I think setting `user_friendly_names` this way is not sufficiently fail safe.
I propose to set a fallback according to $OS_MASTER_VENDOR here
or - probably easier - alternatively test in the subsequent
`case $OS_MASTER_VENDOR` in a fail safe way
for `user_friendly_names` like:
<pre>
    case $OS_MASTER_VENDOR in
        (SUSE)
            if is_true "$user_friendly_names" ; then
                # Exceptional case for SUSE
                ...
            else
                # Usual case for SUSE
                ...
            fi
        (Fedora)
            if is_false "$user_friendly_names" ; then
                # Exceptional case for Fedora and Red Hat
                ...
            else
                # Usual case for Fedora and Red Hat
                ...
            fi
</pre>
so that in case of an invalid `user_friendly_names` value
the usual case for each dirstribution is used.

Reply:
@jsmeix, As said in my comment, checking `user_friendly_names` make sense only for Fedora based system and maybe Debian (need additional test). As there is no "Exceptional case" for Suse, I do prefer keeping the code as simple as possible.

Reply:
Of course, when there is no such "Exceptional case" for Suse
there is no ned for such a check - but a comment in the code
that there is no such "Exceptional case" for Suse would make
things 100% clear to everybody why there is no such a check.